src/v/kafka/client/cluster.cc

```diff
@@ -122,9 +138,31 @@ ss::future<> cluster::dispatch_and_apply_metadata_updates(
 
     try {
         auto broker = _brokers.any();
-        auto metadata_resp = co_await dispatch_metadata_request(
-          broker, std::move(topics_request_list));
-        co_await apply_metadata(to_metadata_update(std::move(metadata_resp)));
+        auto describe_cluster_version = co_await broker->get_supported_versions(
+          describe_cluster_api::key, _as);
+        metadata_update mu;
+        if (describe_cluster_version) {
+            auto describe_cluster_resp
+              = co_await dispatch_describe_cluster_request(broker);
+            mu = to_metadata_update(std::move(describe_cluster_resp));
+        } else {
+            // If describe cluster isn't supported, then dispatch a metadata
+            // request with no topics at API version 10 in order to get the
+            // cluster level authorized operations
+            auto metadata_resp = co_await dispatch_metadata_request(
+              broker, {}, api_version(10));
+            mu = to_metadata_update(std::move(metadata_resp), false);
```

Comment:
It's not obvious what this `false` is for. Could use named_bool or named the variable.